go/mobile/holder.go

```diff
@@ -18,9 +18,10 @@ type reqClaimStatusHandler struct {
 	BaseUrl string
 }
 
-type reqClaimStatusEvent struct {
+type resClaimStatusHandler struct {
```

Comment:
Why has this been renamed to Handler?  I think Event suits better, as this is the event that the user of the library receives.

Reply:
I think I see the reasoning behind this:
there's the request which you poll, and it generates a response that is sent via an event.  I'll accept it but I'd like to discuss it later :)